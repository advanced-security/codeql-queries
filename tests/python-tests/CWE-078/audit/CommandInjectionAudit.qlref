CWE-078/CommandInjectionAudit.ql