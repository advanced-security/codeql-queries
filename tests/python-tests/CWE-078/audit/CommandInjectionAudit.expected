| cmdi.py:7:17:7:17 | ControlFlowNode for i | Usage of command line |
| cmdi.py:9:17:9:17 | ControlFlowNode for i | Usage of command line |
| cmdi.py:11:17:11:30 | ControlFlowNode for Fstring | Usage of command line |
